runtime/startup_byt.c

```diff
@@ -273,53 +273,92 @@ static char_os * read_section_to_os(int fd, struct exec_trailer *trail,
 
 */
 
+static void do_print_help(void)
+{
+  puts("Usage: ocamlrun [<options>] [--] <executable> [<command-line>]\n"
+       "Options are:\n"
+       "  -b  Set runtime parameter b (detailed exception backtraces)\n"
+       "  -I <dir>  Add <dir> to the list of DLL search directories\n"
+       "  -m  Print the magic number of <executable> and exit\n"
+       "  -M  Print the magic number expected by this runtime and exit\n"
+       "  -p  Print the names of the primitives known to this runtime\n"
+       "  -t  Increments the trace level in debug mode by 1\n"
+       "  -v  Set runtime parameter v=61 (GC event information)\n"
+       "  -version  Print version string and exit\n"
+       "  -vnum  Print short version number and exit\n"
+       "  -help  Display this list of options\n"
+       "  --help  Display this list of options");
```

Comment:
Why no `\n` at the end of this line?  Oh, yes, because you're using this awful `puts` function :-)

Reply:
I'm sure I've submitted a patch in the past where the `\n` was forgotten, but I can't find it!